packages/web/src/common/store/player/sagas.ts

```diff
@@ -175,7 +178,17 @@ export function* watchPlay() {
           { uid: PLAYER_SUBSCRIBER_NAME, id: trackId }
         ])
       )
+
+      if (track.genre === Genre.PODCASTS) {
+        // Make sure that the playback rate is set when playing a podcast
+        const rate = yield* select(getPlaybackRate)
+        audioPlayer.setPlaybackRate(rate)
+      } else if (audioPlayer.getPlaybackRate() !== '1x') {
+        // Reset playback rate when playing a regular track
+        audioPlayer.setPlaybackRate('1x')
```

Comment:
sweet. i wonder if it makes sense to use numbers here instead? what do you think?

Reply:
I like it as string bc of type safety and ease of use around the code. the only place that really needs it as a number if the audio element at the end so I went with the string approach. willing to chat about it and update if we feel that number is better though 👍